gemini_test.py

```diff
@@ -46,6 +46,38 @@ def test_random_load(self):
         prepared_results['end_time'] = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
         self._send_email(prepared_results)
 
+        if result['status'] == 'FAILED':
+            self.fail(result['results'])
+
+    def test_random_load_with_nemesis(self):
+        prepared_results = self._prepare_test_results()
+
+        cmd = self.params.get('gemini_cmd')
+        prepared_results['gemini_cmd'] = cmd
+
+        self.db_cluster.add_nemesis(nemesis=self.get_nemesis_class(),
+                                    tester_obj=self)
+
+        self.log.debug('Start gemini benchmark')
+        prepared_results['start_time'] = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        test_queue = self.run_gemini(cmd=cmd)
+
```

Comment:
We need to add a wait here.
Either fixed sleep or wait for total space.
We don't wan't the nemesis to start too early.

Reply:
@roydahan For that moment i think better to add fixed sleep, for example 15% of test_duration, is it ok?

Reply:
Yes.